examples/mujoco/fetch_her_ddpg.py

```diff
@@ -96,6 +95,8 @@ def test_ddpg(args=get_args()):
 
     # logger
     if args.logger == "wandb":
+        import wandb
```

Comment:
If you do it here, you need to do it everywhere. I thought wandb was not an optional dependency, am I wrong?

Reply:
I checked, it's the only example where wandb is import. I also thought about removing it since wandb currently is only part of the 'dev' group dependencies

Reply:
Remove pls